DataSetExplorer/Core/DataSets/CaDETToDataSetProjectBuilder.cs

```diff
@@ -26,6 +26,7 @@ internal class CaDETToDataSetProjectBuilder
         private CaDETMemberType[] _acceptedMemberTypes = {CaDETMemberType.Constructor, CaDETMemberType.Method};
         private readonly List<CodeSmell> _codeSmells;
         private readonly InstanceFilter _instanceFilter;
+        private Dictionary<CaDETClass, Dictionary<CaDETClass, int>> _classReferences = new Dictionary<CaDETClass, Dictionary<CaDETClass, int>>();
```

Comment:
Maybe Dictionary<CaDETClass, List<CoupledClassStrength>> where CoupledClassStrength contains CaDETClass and int?

The _classReference is a bit generic, but I'm not sure what a more suitable name looks like (maybe _classCouplings?)